pytest_fixtures/component/puppet.py

```diff
@@ -36,13 +23,23 @@
 )
 
 
+def enable_puppet(sat_obj):
```

Comment:
You could move this under `satellite_mixins` or under satellite class directly. and use as `sat_object.enable_puppet` 

OR 

Check if we do have it already !